modules/functions-list-kn.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies
+
+// * /serverless/cli_reference/kn-func-ref.adoc
+
+[id="functions-list-kn_{context}"]
+= Listing existing functions
+
+You can list existing functions by using `kn func list`. If you want to list functions that have been deployed as Knative services, you can also use `kn service list`.
+
+[NOTE]
+====
+If the namespace is not specified for the following commands, this defaults to the current namespace.
+====
+
+.Procedure
+
+. List existing functions:
```

Comment:
Should these be bullets instead of numbered, since it's more of an either/or, as opposed to a sequential set of steps?